syntax = "proto3";

package messages;

import "proto/structs.proto";

message ProbingRequest {}

message ProbingReply {
    uint64 currentTime = 1; // milliseconds
    ReplyCode code = 2;
}

service Prober {
    rpc probing(ProbingRequest) returns (ProbingReply) {}
}